cpp/src/utilities/cython.cu

```diff
@@ -895,7 +896,7 @@ call_random_walks(raft::handle_t const& handle,
       detail::create_graph<vertex_t, edge_t, weight_t, false, false>(handle, graph_container);
 
     auto triplet = cugraph::experimental::random_walks(
-      handle, graph->view(), ptr_start_set, num_paths, max_depth);
+      handle, graph->view(), ptr_start_set, num_paths, max_depth, use_padding);
```

Comment:
I might be missing something but is there a reason as to why use_padding is passed as parameter here and not in line 879~880? I see it is only passed when ` weight_t = double`. I was just wondering why

Reply:
fixed